src/gwt/src/org/rstudio/core/client/BrowseCap.java

```diff
@@ -259,6 +263,12 @@ private static final boolean getFixedUbuntuMono()
    {
       Document.get().getBody().addClassName(OPERATING_SYSTEM);
 
+      // Support for scoping styles for macOS BigSur (10.16)
+      if (isMacintoshDesktopBigSur())
+      {
+         Document.get().getBody().addClassName("macos-bigsur");
```

Comment:
Should we wrap this in a Qt version check so if we succeed in Qt 5.15 migration we don't have to remember to undo it?

Don't know of any existing Java/GWT-side checking like this, but presumably could go off the Chromium version; it's Chrome/69 in Qt 5.12.8, Chome/80 in 5.15.

Reply:
I think we should just merge this into your branch and then immediately git revert it there, which will also keep us from having dead code after the update.